dvmconsole/MainWindow.xaml

```diff
@@ -53,6 +53,7 @@
                 <MenuItem Header="_Lock Widgets" IsCheckable="True" Checked="ToggleLockWidgets_Click" Unchecked="ToggleLockWidgets_Click" x:Name="menuToggleLockWidgets" />
                 <MenuItem Header="_Fit Channel Display to Window Size" Click="ResizeCanvasToWindow_Click" x:Name="menuFitToWindow" />
                 <MenuItem Header="_Snap Call History To Window" IsCheckable="True" Checked="ToggleSnapCallHistory_Click" Unchecked="ToggleSnapCallHistory_Click" x:Name="menuSnapCallHistory" />
+                <MenuItem Header="_Stay on Top" IsCheckable="True" Checked="ToggleStayOnTop_Click" Unchecked="ToggleStayOnTop_Click" x:Name="menuStayOnTop" />
```

Comment:
- Relabel Stay on Top to be Always On Top
- Add a <Separator /> between Snap Call History To Window and Always On Top